lib/afterbanks_psd2/base.rb

```diff
@@ -11,11 +11,11 @@ def configure
       yield(configuration)
     end
 
-    def api_call(method:, path:, params: {})
+    def api_call(method:, path:, params: {}, options: {})
```

Comment:
I think it's dangerous allowing some `options` without controlling its content, specially since we then forward these to `RestClient::Request.execute`.

In my opinion, either we just hardcode the `timeout` inside that call (like we do with the other Afterbanks API wrapper) or we allow for an optional `timeout` argument here that we use explicitly if given.

Reply:
Ok i choose the second option : allow for `timeout` into options parameters